src/server/system_services/pool_controllers.js

```diff
@@ -69,7 +64,7 @@ class ManagedStatefulSetPoolController extends PoolController {
             JSON.stringify(pool_k8s_conf, null, 2));
 
         await kube_utils.apply_conf(pool_k8s_conf);
-        return yaml_utils.stringify(pool_k8s_conf);
+        return yaml.stringify(pool_k8s_conf, 6, 2);
```

Comment:
This code should use the kube_store and not kube_utils. we do not want to continue to run kubectl as a process. Especially after we introduced the kube_store to handle communication with the kube API directly